### PR TITLE
Submit proposal for TencentCloud SIG

### PR DESCRIPTION
**Pull request by shengyu**
Proposing SIG for Tencent Cloud provider.
https://github.com/spinnaker/governance/issues/128

### CODE REVIEW

File: sig-tencentcloud/README.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+This SIG focuses on the current TencentCloud provider for Spinnaker.
+
+## Meetings
+
+* Regular SIG Meeting
+  * Last Thursday of every month, at 11 AM Asia/Shanghai ([See in your time zone](https://www.thetimezoneconverter.com/?t=11am&tz=Shanghai))
+  
+  [Agenda](https://docs.google.com/document/d/1BKBzlLh6k5TVi-fEjKy0be3TQtduGgiA03_-g9O7jwM/edit?usp=sharing)
+  [Agenda (qq-docs)](https://docs.qq.com/doc/DZnVDVW1XbURtcnhm)
```
Comment (plumpy) [inline sig-tencentcloud/README.md:11]: Probably just having one agenda doc is enough?

Comment (robzienert) [reply sig-tencentcloud/README.md:11]: I assumed this is because Google Docs aren't available everywhere?

Comment (plumpy) [reply sig-tencentcloud/README.md:11]: But it seems like qq.com is (I can see the linked doc, anyway) so probably just having a single doc is better?

**Comment by robzienert** [comment]
3/5 TOC - merging.

**Pull request merged**